modules/golang/golang

```diff
@@ -1,5 +1,11 @@
 compdef _golang go
 
+function _deps(){
+	app="go"
+
+	$(hash $app 2>/dev/null) || $(myzsh error "Couldn't find application ${app}" && return 1)
```

Comment:
After poking at it, it will not work without a () or $().  In light of the difference between the two, $() seems like the right thing.